README.md

```diff
@@ -99,3 +99,34 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+## Requirements
+
+These are the requirements for this API.
+
+**Note:** The original whitepaper on *zold* can be found [here](https://www.zold.io/). The whitepaper and the documentation found at www.zold.io serve as the highest authority on the subject of *zold*. The following requirements are condensed versions of the points expressed in the aforementioned docs as they relate to the scope of this project:
+
+* Maintain a wallet in structured textual format, within which is a ledger that contains transactions for that wallet.
+* Make payments:
+  * Taxes according to fixed formula
+  * Payments to other wallets
+  * Sign with RSA key
+  * Push to network
+* Receive payments:
+  * Pull the paying wallet from the network
+  * Merge the copies of the paying wallet with our own copy
+* We need to refresh our local database of network nodes by querying the network.
+* We must implement with Java 8
+* Non-functional requirements were not made, but we expect
+  * Flawless concurrency
```

Comment:
@llorllale I would add here that the design of classes should resemble the classes we already have in Ruby project

Reply:
@yegor256 ok